### ISSUE TITLE
Features request

### ISSUE THREAD

**Issue by Subbarao6338** [libre-tube/LibreTube#26]
- video quality selection based on network
- import and export of YouTube subscription
- import and export of YouTube playlist
- playlists and channel download
- download only audio option
- show only video name, when click on video name it will show video description

**Comment by rimthekid** [comment]
> video quality selection based on network

HLS is doing exactly this

> import and export of YouTube subscription
import and export of YouTube playlist
download only audio option
show only video name, when click on video name it will show video description
download with embedded subtitles

added to projects (the last one is a bit tricky maybe not in near future)

> playlists and channel download

don't really know what u mean by that

**Comment by Subbarao6338** [comment]
Playlist and chanel download
- - Means
Download all or selected videos from playlists and chanels.

**Comment by shom13** [comment]
Please add import subscriptions from newpipe or youtube. Thank you

**Comment by rimthekid** [comment]
how do you import from youtube? by signing in to google?

**Comment by Subbarao6338** [comment]
> how do you import from youtube? by signing in to google?

Like newpipe

**Comment by shom13** [comment]
For me i have my subscriptions downloaded through google takeout, i need to import that file. But on general through login to google or import from file.

**Comment by shom13** [comment]
Would be great if you could add import subscription from file in .json format for now

**Comment by rimthekid** [comment]
Yes I was thinking about the json only because I don't want any of that google stuff in the app

**Comment by shom13** [comment]
Thats quite alright, as long as poeple can import their subscriptions, i dont think anybody would complain
